deploy-java/action.yml

```diff
@@ -138,6 +138,19 @@ runs:
         jiraUsername: ${{ inputs.jiraUsername }}
         jiraPassword: ${{ inputs.jiraPassword }}
 
+    - name: On success notify okay to test
+      uses: actions/github-script@v6
+      if: ${{ (inputs.environment !='Production') }}
+      with:
+        github-token: ${{ secrets.PERSONAL_ACCESS_TOKEN }}
+        script: |
+          const result = await github.rest.actions.createWorkflowDispatch({
+            owner: 'cartoncloud',
+            repo: 'api-functional-tests',
+            workflow_id: 'run-tests.yml',
+            environment: ${{ inputs.environment }}
```

Comment:
The environment names don't currently match. i.e. it is `QA 01` for the java service and `qa1` on the API tests.
https://github.com/cartoncloud/api-functional-tests/tree/main/src/test/resources/environments.

Either need to update API test environment names or have some kind of mapping here.

Reply:
Haha, I pointed it out too.